### PR TITLE
update our Dokany drivers

### PR DESCRIPTION
**Pull request by zanderz**
let's see what happens
@strib @taruti

### CODE REVIEW

**Comment by taruti** [comment]
Ping me if you need help with the appveyor failure.

**Comment by strib** [comment]
@taruti yes please, we need your help!  Dokan doesn't seem to support renaming/deleting a file while there's an open file handle to it anymore.  Not sure if that's intentional, and we should delete these tests, or if it's an unintended regression.  I couldn't find anything obvious in the changelog.

**Comment by taruti** [comment]
Once tests pass for #1096 I'll merge it. Please rebase after that and then the libdokan failure should be gone.

File: appveyor.yml [hunk 1]
```diff
@@ -15,18 +15,18 @@ install:
...
   - Bcdedit.exe -set TESTSIGNING ON
   - ps: Restart-Computer -Force
   - ps: Start-Sleep -s 10
-  - cd C:\dokan\Win32\Release
-  - dokanctl /i a
-  - sc create dokan1 type= kernel binPath= system32\drivers\dokan1.sys
-  - sc start dokan1
+  - cd C:\dokany\Win32\Release
+  - dokanctl /i d
+  #- sc create dokan1 type= kernel binPath= system32\drivers\dokan1.sys
```
Comment (strib) [inline appveyor.yml]: Any reason not to remove these lines if they're not needed?

**Pull request merged**